vars/fcosKolaTestIso.groovy

```diff
@@ -0,0 +1,28 @@
+// Run kola testiso
+// Available parameters:
+//     cosaDir:        string  -- cosa working directory
+//     extraArgs:      string  -- extra arguments to pass to `kola testiso`
+//     skipMetal4k:    boolean -- skip metal4k image
+def call(params = [:]) {
+    stage("Test Live Images") {
+        def cosaDir = utils.getCosaDir(params)
+        def extraArgs = params.get('extraArgs', "");
+
+        try {
+            parallel metal: {
+                shwrap("cd ${cosaDir}  && env TMPDIR=\$(pwd)/tmp/ kola testiso -S ${extraArgs} --output-dir tmp/kola-testiso-metal")
```

Comment:
Why do we need the `TMPDIR` again?

Reply:
Meh, looks in my tests we don't really need it at all indeed, checked the other CIs and they are not using it. I was trying to keep it as similar as possible from what we have in cosa ci for now.